data/gui/screens/help1.stkgui

```diff
@@ -35,12 +35,12 @@
                 </div>
                 <spacer width="5" height="1%"/>
 
-                <div width="100%" proportion="5" layout="horizontal-row">
+                <div width="100%" proportion="4" layout="horizontal-row">
                     <icon align="center" width="64" height="64" icon="gui/icons/gift.png"/>
                     <spacer width="25" height="25"/>
                     <bubble proportion="7" height="100%" word_wrap="true"
                             I18N="In the help menu"
-                            text="Collect blue gift boxes, they will give you powerups."/>
+                            text="Collect gift boxes, they will give you powerups."/>
```

Comment:
Your point maybe is that gift boxes are only blue. But the fact that they are blue is an information. If it's in relation with your christmas suggestion, it would make more sense to do that kind of edit if and when it is merged, no ?